src/scheduler/job_info.cpp

```diff
@@ -125,6 +125,7 @@
 #include "server_info.h"
 #include "attribute.h"
 #include "multi_threading.h"
+#include "libpbs.h"
```

Comment:
why did you include this file?

Reply:
As per the latest changes this header file libpbs.h inclusion is not needed. I have removed it in the latest commit. Please check.